test/HeywoodHealthcareTest.js

```diff
@@ -0,0 +1,134 @@
+         */
+        expect(
+            moment(results.individualLocationData[0].timestamp).isBetween(
+                beforeTime,
+                afterTime
+            )
+        );
+
+        // console.log(`${JSON.stringify(results)}`);
+
+        if (process.env.DEVELOPMENT) {
+            file.write(
+                `${process.cwd()}/out_no_browser.json`,
+                `${JSON.stringify(results, null, "   ")}`
+            );
+        }
+    });
+
+    it("should show several slots", async function () {
+        const results = await scraper(false, availabilityFetchService);
+
+        const afterTime = moment();
+
+        const expectedAvailability = true;
+        expect(results.individualLocationData[0].hasAvailability).is.deep.equal(
+            expectedAvailability
+        );
+
+        const date = Object.keys(
+            results.individualLocationData[0].availability
+        )[0];
+
+        expect(
+            results.individualLocationData[0].availability[date]
+                .numberAvailableAppointments
+        ).equals(116);
+        /*
+        Structure conformance expectations:
+
+        - All the timestamps are expected to be between before
+            and after when the scraper was executed
+        - Each site's results object must have a property named "hasAvailability"
+         */
+        expect(
+            moment(results.individualLocationData[0].timestamp).isBetween(
+                beforeTime,
+                afterTime
+            )
+        );
+
+        // console.log(`${JSON.stringify(results)}`);
+
+        if (process.env.DEVELOPMENT) {
+            file.write(
+                `${process.cwd()}/out_no_browser.json`,
+                `${JSON.stringify(results, null, "   ")}`
+            );
+        }
+    });
+});
+
+/* utilities */
+
+/** Generator to feed filenames sequentially to the "with availability" test. */
+function* noAvailabilityFilenames() {
+    yield "firstPage-noSlots-hasMoreTimes-scripts-removed.html";
+    yield "secondPage-noSlots-scripts-removed.html";
+}
+const noAvailabilityFilenameGenerator = noAvailabilityFilenames();
+
+const noAvailabilityFetchService = {
+    async fetchAvailability(/*site*/) {
+        return loadTestHtmlFromFile(
+            "HeywoodHealthcare",
+            noAvailabilityFilenameGenerator.next().value
+        );
+    },
+};
+
+/** Generator to feed filenames sequentially to the "with availability" test. */
+function* availabilityFilenames() {
+    yield "firstPage-noSlots-hasMoreTimes-scripts-removed.html";
+    yield "secondPage-lots0fSlots-scripts-removed.html";
+    yield "thirdPage-someSlots-noMoreTimes-scripts-removed.html";
+}
+const availabilityFilenameGenerator = availabilityFilenames();
+
+const availabilityFetchService = {
+    async fetchAvailability(/*site*/) {
+        return loadTestHtmlFromFile(
+            "HeywoodHealthcare",
+            availabilityFilenameGenerator.next().value
```

Comment:
`testFolderName` should be `"HeywoodHealthCare"`.  **(capitalize the C)**.

Reply:
same as above... renaming the directory will fix this too

Reply:
Hmm, I just renamed the folder to HeywoodHealthcare in my working folder, but that doesn't seem to be sensed as a change by either VS Code or Git Desktop. Any suggestions as to how to get one or both to see the change? Or is there a command-line way of forcing this change?

Reply:
Solved it by introducing a spelling change, committing, changing to the desired spelling and case, and committing again.